burn-book/src/building-blocks/tensor.md

```diff
@@ -285,6 +285,7 @@ Those operations are only available for `Int` tensors.
 | ------------------------------------------------ | ------------------------------------------------------- |
 | `tensor.arange(5..10, device)       `            | `tensor.arange(start=5, end=10, device=device)`         |
 | `tensor.arange_step(5..10, 2, device)`           | `tensor.arange(start=5, end=10, step=2, device=device)` |
+| `tensor.indices(shape, device)`                  | `torch.meshgrid(tensors)`                               |
```

Comment:
Two notes:

1. Usage should be `Tensor::indices(shape, device)` like `Tensor::cat` or `Tensor::empty`.
2. Given the implementation here, there is no 1-to-1 equivalent for torch since meshgrid takes tensors as input and not the shape of the desired grid. In this case this is much closer to `numpy.indices` but the indexing is in cartesian space. The actual torch equivalent for the 2D example in your unit tests would be:
```py
yv, xv = torch.meshgrid([torch.arange(2), torch.arange(2)], indexing='xy')
grid = torch.stack((xv, yv), 2)
```

So in this case, I'm not sure we would have to provide the comparison in the table.